src/styles/ForecastDetails.css

```diff
@@ -0,0 +1,11 @@
+.forecast-details>* {
+  padding-block-start: 10px;
+  font-weight: 600;
+  text-align: center;
+  font-size: 1.5rem;
+}
+
+.forecast-details__temperature>*:first-of-type {
+  padding-block-end: 10px;
+  margin-block-start: -20px;
```

Comment:
I would definitely avoid negative margins unless absolutely necessary and there aren't many genuine use cases. Here, I could achieve the exact same time by replacing the margin with:
`padding-top: 20px`